src/library/scala/runtime/ScalaRunTime.scala

```diff
@@ -139,6 +139,19 @@ object ScalaRunTime {
   // More background at ticket #2318.
   def ensureAccessible(m: JMethod): JMethod = scala.reflect.ensureAccessible(m)
 
+  def lookupStructuralMethod(c: Class[_], m: String, params: Array[Class[_]]): JMethod = {
```

Comment:
This doesn't pass MiMa. It's more than an internal issue: if we compile new 2.12 code to use this method, it won't run when an older 2.12 library is on the classpath.